### ISSUE TITLE
Scammed by xt.com

### ISSUE THREAD

**Issue by Roman228888** [ccxt/ccxt#18346]
### Description

I was scammed by xt.com. All whom they have offended, let's collect a collective question. My email: arubikk@gmail.com attach all information in pdf file. I will write to Interpol, financial regulators, look for lawyers. We need to be together. Together - we are force !

### Code

```
  

```

**Comment by kroitor** [comment]
Duplicate post here: https://github.com/ccxt/ccxt/issues/18315 (please move this discussion there, thank you).

**Comment by Roman228888** [comment]
https://t.me/+CZHyckLfHBc1YWNi

Scammed by xt.com

**Comment by kroitor** [comment]
> https://t.me/+CZHyckLfHBc1YWNi
> 
> Scammed by xt.com

@Roman228888 please post your story here: https://github.com/ccxt/ccxt/issues/18315

**Comment by Roman228888** [comment]
In 2021 year i earning on xt 242 000$ , and xt roll back order, before xt frozen my account 5000$ . Now xt frozen my account 3700$. Abnormal transaction. Its my work strategy. All my orders on ubnormal price! This company scammers !
/
[XT IJZ Trade on Nov 26 2021.pdf](https://github.com/ccxt/ccxt/files/11884005/XT.IJZ.Trade.on.Nov.26.2021.pdf)

**Comment by yemreak** [comment]
> In 2021 year i earning on xt 242 000$ , and xt roll back order, before xt frozen my account 5000$ . Now xt frozen my account 3700$. Abnormal transaction. Its my work strategy. All my orders on ubnormal price! This company scammers !
> /
> [XT IJZ Trade on Nov 26 2021.pdf](https://github.com/ccxt/ccxt/files/11884005/XT.IJZ.Trade.on.Nov.26.2021.pdf)

@Roman228888  can u post it to #18315 too

**Comment by android-programmer** [comment]
HI guys, I tried to do withdrawal less than 50k (their website says lower than 50k does not require KYC) , but when I withdrew, XT responded to me an email that I need to do KYC. But when I submitted my KYC documents , XT responded to me the country I am not in is not allowed.  So they are trying every excuses to hold your money. 
Is there anyone can advise what to do ? Thanks guys ! @Roman228888 @kroitor @yemreak
